NTupleAnalyzer_nanoAOD.py

```diff
@@ -65,7 +70,7 @@
 
 #Switches to add BDT discriminants
```

Comment:
what exactly does it mean to 'add BDT discriminants'? To run the BDT training? Or add them to the root file output? Why would you not want them?

Reply:
Adding means add the weights of the already trained BDTs to the root file. This can add considerable time to running, so if we don't need them, I find it's better to not include them (like the systematics). Right now we are producing trees to use in training the BDT with all three years so we don't need to add any BDT weights for this purpose. Speed is the priority right now. After training, we will rerun the analyzer with systematics and BDT weights added in.

Reply:
👍